nimbleclient/v1/restclient.py

```diff
@@ -365,3 +415,16 @@ def perform_bulk_resource_action(self, resource, action, **params):
 
         resp = self.post(f"{self._ENDPOINTS[resource]}/actions/{action}", **params)
         return resp['data'] if 'data' in resp else resp
+
+    def build_advanced_criteria(self, filter):
```

Comment:
This should be a class method and `filter` should really be split into two separate parameters:
```python
@classmethod
def build_advanced_criteria(cls, operator, criteria):
```

Reply:
Agreed. Fixed accordingly